CMakeLists.txt

```diff
@@ -72,6 +72,13 @@ find_package(Ruby)
 if(NOT Ruby_FOUND)
   message(STATUS "Ruby tests disabled: Ruby interpreter not found.")
 else()
+  execute_process(COMMAND ${RUBY_EXECUTABLE} -rffi-e "puts FFI"
```

Comment:
Are you missing a space here?
Looks like it should be -rffi-e

Reply:
Yes, good spot.  Thanks